### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by louismillsdiffblue**
Hi,

I've analysed your codebase and produced some unit tests for the following classes - Formats and Utils

I've written the tests for these functions with the help of [Diffblue](https://www.diffblue.com/) [Cover](https://www.diffblue.com/s/Diffblue-Cover-Datasheet.pdf).

Hopefully, these tests should help you detect regressions caused by future code changes.

### CODE REVIEW

File: vjtop/pom.xml [hunk 1]
```diff
@@ -31,6 +31,17 @@
...
 			<systemPath>${toolsjar}</systemPath>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.8.2</version>
+			<scope>test</scope>
+		</dependency>
+        <dependency>
+            <groupId>com.diffblue</groupId>
```
Comment (sudo-jaa) [inline vjtop/pom.xml]: Looks like we don't need to pull in Deeptestutils for this project.

File: vjtop/src/test/java/com/vip/vjtools/vjtop/util/FormatsTest.java [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+        final LongWarning warning = null;
+
+        // Act
+        thrown.expect(NullPointerException.class);
+        Formats.toSizeUnitWithColor(size, warning);
+
+        // Method is not expected to return due to exception thrown
+    }
+
+}
```
Comment (sudo-jaa) [inline vjtop/src/test/java/com/vip/vjtools/vjtop/util/FormatsTest.java]: Add a newline here :+1:

**Review by sudo-jaa — APPROVED** [review]

File: vjtop/src/test/java/com/vip/vjtools/vjtop/util/FormatsTest.java [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+import com.vip.vjtools.vjtop.WarningRule.LongWarning;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import java.util.ArrayList;
+
+public class FormatsTest {
+
```
Comment (petersear) [inline vjtop/src/test/java/com/vip/vjtools/vjtop/util/FormatsTest.java:11]: Remove one blank line

File: vjtop/src/test/java/com/vip/vjtools/vjtop/util/FormatsTest.java [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+        final int length = 1;
+        final int rightLength = 6;
+
+        // Act
+        final String retval = Formats.shortName(str, length, rightLength);
+
+        // Assert result
+        Assert.assertEquals("!", retval);
+    }
+
```
Comment (petersear) [inline vjtop/src/test/java/com/vip/vjtools/vjtop/util/FormatsTest.java:113]: Remove one blank line

**Review by petersear — CHANGES_REQUESTED** [review]

**Review by petersear — APPROVED** [review]

**Review by paulkennethkent — APPROVED** [review]

**Review by JohnLBergqvist — CHANGES_REQUESTED** [review]
Your commit messages must be in the following format (commit messages can be multi-line):
```
Add unit tests for <class>
These tests were written by Diffblue Cover.
Dependencies required to run the tests have also been added.
```

Secondly, the title of the PR should contain the classes you are modifying, or at least be like this:
`Add unit tests for com.vip.vjtools.vjtop.util`

**Pull request merged**